test/integration-new/cni/pod_traffic_test_PD_toggle.go

```diff
@@ -42,10 +42,6 @@ var _ = Describe("Test pod networking with prefix delegation enabled <-> disable
 	JustBeforeEach(func() {
 		// TODO Gingko doesnt support beforeAll so while adding upgrades/downgrades will move this to a suite
 		if firstRun {
-			By("creating test namespace")
```

Comment:
Any reason we are removing this? Is the namespace carried over from the prior test suite?

Reply:
The suite had the namespace creation/deletion being done in Before/AftureSuite blocks. This change introduced by PD tests creates/deletes the same namespace in JustBeforeEach/JustAfterEach blocks. 

If the PD test runs first and delete the namespace, then the other tests would fail which previously relied on namespace to be present during test.  

I think we can either remove this, or change the PD test namespace to fix the issue.